### PR TITLE
Include notebook imagestream manifests to notebook repos

### PR DESCRIPTION
**Pull request by harshad16**
Include notebook imagestream manifests to notebook repos

## Description

This is to reference notebook imagestream rates. 
Related-to: https://github.com/opendatahub-io/notebooks/issues/185

Copied from: https://github.com/opendatahub-io/odh-manifests/tree/master/notebook-images

## How Has This Been Tested?

compare the https://github.com/opendatahub-io/odh-manifests/tree/master/notebook-images and the files changed.

## Merge criteria:

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: .github/workflows/notebooks-digest-updater-upstream.yaml [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# The aim of this GitHub workflow is to update the params.env file with the latest digest.
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        required: true
+        description: "Provide the name of the branch you want to update ex main, vx.xx etc"
```
Comment (atheo89) [inline .github/workflows/notebooks-digest-updater-upstream.yaml:7]: A small change. Could we consider updating the notation from `vx.xx` to `YYYYx` in the description to align it with the repository's structure?

**Review by atheo89 — CHANGES_REQUESTED** [review]

**Comment by atheo89** [comment]
Thank you for integrating this into the notebooks repository and implementing the recommended adjustments! This brings us one step closer to addressing the remaining item on https://github.com/opendatahub-io/architecture-decision-records/pull/16.

Regarding the failing tests, it's worth noting that they are not directly linked to the changes made in this PR. Instead, they are a consequence of an unmerged PR (trusty) and a known issue related to the Anaconda notebook. Given this, I recommend proceeding with the merging of this specific PR.

/lgtm

**Comment by harshad16** [comment]
Thanks for the details about the CI.
Merging this in manually for now.

**Comment by zdtsw** [comment]
does it mean I can mark 

Workbenches | opendatahub-io/notebooks | main | Yes
-- | -- | -- | --

in [ADR](https://github.com/opendatahub-io/architecture-decision-records/pull/16)

**Comment by harshad16** [comment]
@zdtsw yes correct, now we have the manifests in the repo.
I will comment on the ADR as well.

**Pull request merged**